app/src/main/java/com/example/internetshop/presentation/activity/fragments/FavoriteListFragment.kt

```diff
@@ -33,6 +34,18 @@ class FavoriteListFragment: BaseFragment() {
         component.inject(this)
     }
 
+    override fun getTitle(): String {
```

Comment:
All three methods should be in body style

Reply:
fixed